### DISCUSSION TITLE
通知如何关闭

### DISCUSSION THREAD

**Discussion by xinhunshang** [LSPosed/LSPosed#2343] [Q&A] [answered]
![Screenshot_2023-01-17-18-23-40-575_com android browser](https://user-images.githubusercontent.com/50572484/212874432-e1505d7f-1f89-49d9-840c-0e4e331733b3.jpg)
![Screenshot_2023-01-17-18-24-50-553_com android shell](https://user-images.githubusercontent.com/50572484/212874438-68d00a17-1f6c-4864-9135-ceca29f751a4.jpg)

**Comment by yujincheng08** [comment, marked as answer]
这都看不见？
![image](https://user-images.githubusercontent.com/5022927/212874629-78092d57-e1da-464b-89df-954b2ea8140a.png)

**Reply by TomasJohnny** [reply]
hhh
